app/resources/freetextsearch.py

```diff
@@ -59,9 +59,14 @@ def get(self):
         start_time = time.time()
         kwargs = self.parser.parse_args()
         kwargs.pop('size');
-        kwargs['fields'] = ['id', 'approved_symbol']; #do not want any other fields
+       
+        
         
         filter = kwargs.pop('filter') or ['all']
```

Comment:
`filter` is a reserved word in python, it is common practice to append an `_` and in this case call the variable `filter_`, or you can be more creative than me.